mmedit/datasets/basic_conditional_dataset.py

```diff
@@ -156,6 +184,9 @@ def load_data_list(self):
 
         if not self.ann_file:
             samples = self._find_samples(file_client)
+        elif self.ann_file.endswith('json'):
+            samples = mmengine.fileio.io.load(self.ann_file)
+            samples = [[name, label] for name, label in samples.items()]
         else:
             lines = mmengine.list_from_file(self.ann_file)
```

Comment:
What if a user put into a yaml ann_file? Maybe we can force file suffix to `json` and `txt`.